recipes/leptonica/all/conandata.yml

```diff
@@ -1,4 +1,4 @@
 sources:
-  "1.78.0":
```

Comment:
AFAIK we have a policy to never remove existing versions to avoid breakage.
@danimtb could you comment on that?

Reply:
Was it uploaded? PR was never accepted.

Reply:
seems so:
```
Remote 'conan-center':
leptonica/1.75.1@bincrafters/stable
leptonica/1.76.0@bincrafters/stable
leptonica/1.78.0
```

Reply:
Oh, my bad, 1.78.0 was already merged. Let me bring it back.